Misc/ACKS

```diff
@@ -1901,3 +1901,4 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
+Robert Mandic
```

Comment:
the list is in rough alphabetical order by last names.

Reply:
Fixed 72ccdac3408a46d834260728b1a6203466e3cbc7